model/patch_lifecycle.go

```diff
@@ -486,6 +495,26 @@ func finalizeChildPatch(ctx context.Context, childPatchId string, requester stri
 	return nil
 }
 
+func SubscribeOnParentOutcome(parentStatus string, p *patch.Patch, requester string) error {
+	parentPatch, err := patch.FindOneId(p.Triggers.ParentPatch)
```

Comment:
why is it necessary to find the parent patch?

Reply:
It's needed here: 
`patchSub := event.NewParentPatchSubscription(parentPatch.Id.Hex(), subscriber)`
But I'm refactoring it and querying for it will no longer be necessary (it can be passed it).